hw_2.js

```diff
@@ -0,0 +1,82 @@
+"use strict";
+//NORMAL level
+
+//task 1
+let x = 20;
+let y = 58;
+let z = 42;
+
+console.log(x + y + z);
+
+//task 2
+let secInMinute = 60;
+let minInHour = 60;
+let hourInDay = 24;
+let dayInYear = 365;
+
+let myAgeInSeconds = 39 * dayInYear * hourInDay * minInHour * secInMinute;
+
+//task 3
+let count = 42;
+let userName = '42';
+
+let my_let1_variant1 = String(count);
+let my_let1_variant2 = `${count}`;
+
+let my_let2_variant1 = Number(userName);
+let my_let2_variant2 = +userName;
+
+//task 4
+let a = 1;
+let b = 2;
+let c = 'белых медведей';
+
+console.log(`${a}` +  `${b}` + ' ' + c);
```

Comment:
нету смысла тут писать каждую переменную через ``
```suggestion
console.log(a +  b + ' ' + c);
```